network-api/networkapi/campaign/views.py

```diff
@@ -148,26 +148,12 @@ def signup_submission(request, signup):
     if cid is not None and cid != '':
         data['campaign_id'] = cid
 
-    # If we want to subscribe them through basket, do so, if not, subscribe them using SQS.
-    if settings.MOFO_NEWSLETTER_SUBSCRIBE_METHOD == 'BASKET':
-        response = basket.subscribe(data['email'], data['newsletters'], lang=data['lang'])
-        if response['status'] == 'ok':
-            return JsonResponse(data, status=status.HTTP_201_CREATED)
+    # Subscribing to newsletter using basket.
+    response = basket.subscribe(data['email'], data['newsletters'], lang=data['lang'])
+    if response['status'] == 'ok':
+        return JsonResponse(data, status=status.HTTP_201_CREATED)
 
-        return JsonResponse(data, status=status.HTTP_400_BAD_REQUEST)
-
-    else:
-        # pack up as a basket message
-        message = json.dumps({
-            'app': settings.HEROKU_APP_NAME,
-            'timestamp': datetime.now().isoformat(),
-            'data': {
-                'json': True,
-                'form': data,
-                'event_type': 'newsletter_signup_data'
-            }
-        })
-        return send_to_sqs(crm_sqs['client'], crm_queue_url, message, type='signup')
```

Comment:
does this mean we're no longer directly using sqs? If so, we can probably also clean up that code.

Reply:
Hi @Pomax ! We are still using SQS for petition sign ups [here](https://github.com/mozilla/foundation.mozilla.org/blob/8cb61055409304639ec1f3cb04e1ed8679fd8140/network-api/networkapi/campaign/views.py#L227)

Reply:
I see, in that case let's not delete this function =D